README.md

```diff
@@ -194,8 +194,16 @@ This parameter will not be used to determine if a scheduled task is in sync or n
 
 ##### `compatibility`
 
+This provider feature is only available with the `taskscheduler_api2` provider.
+
 The compatibility level associated with the task.
-May currently only be set to 1 for compatibility with tasks on a Windows XP or Windows Server 2003 computer.
+Can be set to:
+
+- `1` for compatibility with tasks on a Windows XP or Windows Server 2003 computer
+- `2` for compatibility with tasks on a Windows 2008 computer,
+- `3` for compatibility with new features for tasks introduced in Windows 7 and 2008R2
+- `4` for compatibility with new features for tasks introduced in Windows 8, Server 2012R2 and Server 2016
+- `6` for compatibility with new features for tasks introduced in Windows 10
```

Comment:
I think we might want to add a note about "6" being undocumented and that it's YMMV / not recommended.

I'm not really sure if people should be using it or not, it doesn't show up in the WMI MOFs from what I can tell (at least it's not exposed through `wbemtest`), PowerShell can't get at it properly, etc.

We might also want to link to the docs at MSDN that talk about task scheduler compat stuff https://msdn.microsoft.com/en-us/library/windows/desktop/aa384138(v=vs.85).aspx

Reply:
I think this makes good sense and is the right place to include such a note, adding now.

Reply:
Notes added below.